middleware/routes/api.php

```diff
@@ -17,3 +17,39 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::get('/provider_users/{user_id}/courses', [App\Http\Controllers\ProviderUserController::class, 'getProviderUserResources']);
+Route::post('/provider_platforms', [App\Http\Controllers\ProviderPlatformController::class, 'registerProvider']);
+
+// Naming scheme for routes needs to be decided
+
+// get all courses fo all users
+Route::get('/api/courses', [App\Http\Controllers\ProviderUserResourceController::class, 'index']);
+// add course for a specific user
+Route::post('api/users/{user_id}/courses', [App\Http\Controllers\ProviderUserResourceController::class, 'store']);
```

Comment:
users in starfish are going to be organizations, not students. I think these apis should be 'api/students/{student_id}/courses'

Reply:
I'm not saying you need to change this now, just that we need to have a larger discussion about it. I know a lot of things are going to change as we mature the codebase.